### PR TITLE
sync `css values and units` for zh-CN

### LINKED ISSUE #9250: 我觉得这篇文章里面有一段有语义冲突
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Values_and_units

### What specific section or headline is this issue about?

段落意思表述不清，不知道在讲什么

### What information was incorrect, unhelpful, or incomplete?

CSS 中的值是一种定义允许子值集合的方法。**这意味着如果您看到<color>是有效的，那么您就不需要考虑可以使用哪些不同类型的颜色值—关键字、十六进制值、rgb()函数等等**。假设浏览器支持这些可用的<color>值，则可以使用它们任意一个。**MDN 上针对每个值的页面将提供有关浏览器支持的信息。**例如，如果您查看 [<color>](https://developer.mozilla.org/zh-CN/docs/Web/CSS/color_value)的页面，您将看到浏览器兼容性部分列出了不同类型的颜色值以及对它们的支持。

这一段的这两句话给我的理解是不一样的，第一句加粗的语句的意思就好像是只要关键字和16进制、rgb()函数中的一个可以用，其他的值就可以用，而第二句加粗的语句的意思，还是要根据浏览器的支持区选择值。

### What did you expect to see?

更明确的语句吧，这样太摸棱两可了

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/learn/css/building_blocks/values_and_units`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Values_and_units
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/learn/css/building_blocks/values_and_units/index.md
* Last commit: https://github.com/mdn/translated-content/commit/4c181a1a21dce82421acee56d07749bcad37c242
* Document last modified: 2022-10-03T13:46:51.000Z

</details>

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
close #9250

### CODE REVIEW

File: files/zh-cn/learn/css/building_blocks/values_and_units/index.md [hunk 1]
```diff
@@ -49,13 +49,13 @@ h1 {
 }
 ```
 
-CSS 中的值是一种定义允许子值集合的方法。这意味着如果您看到`<color>`是有效的，那么您就不需要考虑可以使用哪些不同类型的颜色值—关键字、十六进制值、`rgb()`函数等等。假设浏览器支持这些可用的`<color>`值，则可以使用它们任意一个。MDN 上针对每个值的页面将提供有关浏览器支持的信息。例如，如果您查看 [`<color>`](/zh-CN/docs/Web/CSS/color_value)的页面，您将看到浏览器兼容性部分列出了不同类型的颜色值以及对它们的支持。
+CSS 中的值类型是一种定义了一些可使用的值的集合的方式。这意味着如果你看到的 `<color>` 是有效的，那么你就不需要在意使用的是哪种类型，不管是关键字、十六进制值还是 `rgb()` 函数等都是有效的。如果浏览器支持这些可用的 `<color>` 值，则可以使用它们当中的任意一个。MDN 上针对每个值类型的页面将提供有关浏览器支持的信息。例如，如果你查看 [`<color>`](/zh-CN/docs/Web/CSS/color_value) 的页面，你将看到浏览器兼容性部分列出了不同类型的颜色值以及对它们的支持。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/values_and_units/index.md:52]:
```suggestion
CSS 中的值类型是一种定义了一些可使用的值的集合的方式。这意味着如果你看到的 `<color>` 是有效的，那么你就不需要考虑可以使用哪种类型——不管是关键字、十六进制值还是 `rgb()` 函数等都是有效的。如果浏览器支持这些可用的 `<color>` 值，则可以使用它们当中的任意一个。MDN 上针对每个值类型的页面将提供有关浏览器支持的信息。例如，如果你查看 [`<color>`](/zh-CN/docs/Web/CSS/color_value) 的页面，你将看到浏览器兼容性部分列出了不同类型的颜色值以及对它们的支持。
```

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Hi @awxiaoxian2020, is this PR ready for merging. We could wait a couple of days for the review from @5z5.

**Comment by awxiaoxian2020** [comment]
> Hi @awxiaoxian2020, is this PR ready for merging. We could wait a couple of days for the review from @5z5.

I originally planned to re-translate this article. But I'm a little busy recently, sorry.

**Comment by yin1999** [comment]
Let's merge this

**Comment by 5z5** [comment]
thanks



| |
***@***.***
|
|
***@***.***
|




---- Replied Message ----
| From | ***@***.***> |
| Date | 11/28/2022 19:44 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [mdn/translated-content] sync `css values and units` for zh-CN (PR #9318) |

Merged #9318 into main.

**Pull request merged**